bridge/engine/EngineMixer.cpp

```diff
@@ -1543,26 +1534,11 @@ void EngineMixer::onSctpMessage(transport::RtcTransport* sender,
     const void* data,
     size_t length)
 {
-    // TODO  parse this with wonderful json parser and figure out if this is a barbell UMM
-    // if it is we should handle it locally in onBarbellUserMediaMap
-    if (_engineBarbells.contains(sender->getEndpointIdHash()))
+    if (EngineBarbell::isFromBarbell(sender->getTag()))
     {
-        if (length == 0)
-        {
-            return;
-        }
         auto packet = webrtc::makeUniquePacket(streamId, payloadProtocol, data, length, _sendAllocator);
```

Comment:
send all barbell data stream message directly to EngineMixer. Do not pass them through MixerManager.